misc/utils.h

```diff
@@ -14,10 +14,14 @@ std::string JsonStringToPbBinary(const std::string &table_name, const std::strin
 
 #endif
 
+#ifndef SWIG
+
 extern "C" {
 
 size_t TableNameToTypeUrl(const char *table_name, char *type_url, size_t buffer_size);
```

Comment:
>TableNameToTypeUrl [](http://example.com/codeflow?start=7&length=18)

Why you need C functions?

Reply:
For the detection of autoconf.
Since the mangled name of C++ interface, autoconf isn't easy to detect the required API for adapting the library version.

Reply:
The concern is that this duplication declarations are not sclable.